tutorials/08-voting/contracts/GovernanceToken.cdc

```diff
@@ -0,0 +1,200 @@
+import FungibleToken from "./FungibleToken.cdc"
+
+// GovernanceToken.cdc
+//
+// The GovernanceToken contract is a sample implementation of a fungible token on Flow which can be used for voting.
+
+pub contract GovernanceToken: FungibleToken {
```

Comment:
I hesitate creating a tutorial that implements the Fungible Token standard, because in the user's learning journey, we haven't actually explained how the fungible token standard works yet, so it might be introducing more complexity than is worth introducing. What do you think?

Reply:
I got the idea to use the Fungible Token contract because Satyam and you were mentioning it in the issue discussion. 
I don't really have a strong opinion here.